tests/test-helpers/app4-both/R/upper.R

```diff
@@ -0,0 +1 @@
+upperHelper <- 'abc'
```

Comment:
Is this left over from running this? https://github.com/rstudio/shiny/blob/e84beffee39d540d3e891faafb9f333af32deca4/tests/testthat/test-app.R#L54

If it is, then that test block should have an `on.exit` that removes the file. (It's OK to put it in this PR)

Reply:
I went with ye olde `testthat::teardown()` for this instead of `on.exit()`